### ISSUE TITLE
Voxelization issues for certain fuel tanks

### ISSUE THREAD

**Issue by mjn33** [ferram4/Ferram-Aerospace-Research#93]
I've noticed that the Rockomax fuel tanks seem to have voxelization issues. Starting from a new sandbox game on a fresh KSP 1.0.4 installation using FAR at df3d535e66a74d7646cd89d5b5f2dd6ecb24a407, the following screenshots demonstrate the problem:

[screenshot0.png](https://dl.dropboxusercontent.com/u/15422248/farbug6/screenshot0.png)
[screenshot1.png](https://dl.dropboxusercontent.com/u/15422248/farbug6/screenshot1.png)

I did see a forum post with a screenshot showing a [similar problem](http://forum.kerbalspaceprogram.com/threads/20451-1-0-3-Ferram-Aerospace-Research-v0-15-3-1-Garabedian-6-22-15?p=2037342&viewfull=1#post2037342) (in the second screenshot, notice the bump between the Rockomax tank and the engine). It isn't noticeable on other fuel tanks with the moderate area ruling setting, but can be observed on some other tanks using strict area ruling (albeit much less noticeably). 

[screenshot2.png](https://dl.dropboxusercontent.com/u/15422248/farbug6/screenshot2.png)
[screenshot3.png](https://dl.dropboxusercontent.com/u/15422248/farbug6/screenshot3.png)
[screenshot4.png](https://dl.dropboxusercontent.com/u/15422248/farbug6/screenshot4.png)

I checked FAR v0.15.2 "Ferri" on my KSP 1.0.2 backup and verified it was fine using strict area ruling.
[screenshot37.png](https://dl.dropboxusercontent.com/u/15422248/farbug6/screenshot37.png)

[Player.log](https://dl.dropboxusercontent.com/u/15422248/farbug6/Player.log) for good measure

**Comment by ferram4** [comment]
Yes, I know what's causing this.  The Rockomax issues are actually due to the fact that the colliders for them are too short for their size; check out the collision and you can see this.  End result is that they have the worst behavior with respect to this as a result.  The other corner issues are likely some weirdness with the partially-filled voxels that should be addressable, but I'm not sure of the best way to go about it.

**Comment by mjn33** [comment]
Yeah, as a workaround I'm using the following ModuleManager patch in FARVoxelGeometryOverrides.cfg:

```
@PART[fuelTank1-2,fuelTank2-2,fuelTank3-2,fuelTank4-2]:AFTER[FerramAerospaceResearch]
{
	@MODULE[GeometryPartModule]
	{
		%forceUseMeshes = True
	}
}
```

I think the only reason it worked in Ferri was because of another bug fixed in c00e5ea47c2be57a38a43d0f386738a7da5bd865. The other issues I think are just lack of detail in the collider meshes, which I guess are an optimization (hell, I never even noticed the collision mesh problems on those Rockomax tanks until now).

**Comment by mjn33** [comment]
Well, since my main issue has been resolved, I'm going to close this issue. Thanks for the help, Ferram.
